### ISSUE TITLE
Confused by deltaX and deltaY

### ISSUE THREAD

**Issue by lhrinaldi** [FormidableLabs/react-swipeable#157]
Documentation: 

```js
{
  event,          // source event
  deltaX,         // x offset (current.x - initial.x)
  deltaY,         // y offset (current.y - initial.y)
  absX,           // absolute deltaX
  absY,           // absolute deltaY
  velocity,       // √(absX^2 + absY^2) / time
  dir,            // direction of swipe (Left|Right|Up|Down)
}
```

Codebase:

```js
const { clientX, clientY } = event.touches ? event.touches[0] : event
const [x, y] = rotateXYByAngle([clientX, clientY], props.rotationAngle)
const deltaX = state.xy[0] - x
const deltaY = state.xy[1] - y
```

My question here is, are `clientX` and `clientY` the current x and y coordinates of the mouse?
The docs says current.x minus initial.x, I was expecting that a mouse movement from left to right returns a positive value, but it's the opposite.

Am I getting something wrong here?

**Comment by hartzis** [comment]
@r-inaldi Thank you for posting the issue, you've absolutely found a discrepancy. The readme should say `initial - current`, I'll get a small PR to update the docs. At least it looks like we've been consistent between v4 & v5.

I think you're absolutely correct and we should be doing `current - initial` and I think that's why the docs say that, but it looks like we just "copied" what we had in v4 to v5 in the code.

Going to make a note to make this change for `v6` when we drop the `<Swipeable />` component.

**Comment by hartzis** [comment]
@r-inaldi a side note, i'll try to merge in #151 soon which adds `[initialX, initialY]` as `eventData.initial` and that could be used to derive the "correct" delta values you're looking for.

**Comment by r-inaldi** [comment]
Thanks for the fix @hartzis. Wasn't sure if I was getting something wrong, started to really doubt my knowledge about page coordinates. 😂

**Comment by psikoi** [comment]
I believe this should be closed?

**Comment by hartzis** [comment]
@psikoi thanks for the :poke:, i was keeping it open till i merged and deployed #162
